collector-http/src/main/java/zipkin2/collector/otel/http/SpanTranslator.java

```diff
@@ -70,9 +73,18 @@ private static zipkin2.Span generateSpan(Span spanData, InstrumentationScope sco
     long startTimestamp = nanoToMills(spanData.getStartTimeUnixNano());
     long endTimestamp = nanoToMills(spanData.getEndTimeUnixNano());
     Map<String, AnyValue> attributesMap = spanData.getAttributesList().stream().collect(Collectors.toMap(KeyValue::getKey, KeyValue::getValue));
-    zipkin2.Span.Builder spanBuilder = zipkin2.Span.newBuilder()
-        .traceId(OtelEncodingUtils.traceIdFromBytes(spanData.getTraceId().toByteArray()))
-        .id(OtelEncodingUtils.spanIdFromBytes(spanData.getSpanId().toByteArray()))
+    zipkin2.Span.Builder spanBuilder = zipkin2.Span.newBuilder();
+    byte[] traceIdBytes = spanData.getTraceId().toByteArray();
+    long high = bytesToLong(traceIdBytes, 0);
+    if (high == 0) {
+      spanBuilder.traceId(INVALID_TRACE);
```

Comment:
invalid traces should end up being dropped instead I think. You can look at collectormetrics and usage for an example.

Reply:
@codefromthecrypt 

What does "collectormetrics" refer to?

If I create a span with an invalid ID in Zipkin it throws an `IllegalArgumentException`. Does that mean that catch the exception and increment the "drop" counter metric?

Reply:
like this https://github.com/openzipkin/zipkin/blob/91976d83578a4a4bd22f2d3ddf637c36d8a27c54/zipkin-collector/rabbitmq/src/test/java/zipkin2/collector/rabbitmq/ITRabbitMQCollector.java#L113

Reply:
Increment spanDropped if the received span is invalid in https://github.com/openzipkin-contrib/zipkin-otel/pull/13/commits/c869d693ecb37e91bf605bc5546f215379a9a254